### PR TITLE
Adds CC_ref_salinity parameter to DOSTAM cals

### PR DESCRIPTION
**Pull request by s-pearce**
@cwingard Can you please review and merge?
I added the configuration parameter CC_ref_salinity to each DOSTAM calibration csv file with a default value of 0. 
I ended up converting many of the line endings from windows to unix, so many files will reflect that in the changes

@cdobs make sure you agree with these changes.

thanks,
Stuart

### CODE REVIEW

File: calibration/DOSTAM/CGINS-DOSTAM-00364__20180216.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
-serial,name,value,notes
-364,CC_calibration_date,1.518739200000E+12,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_364_Calibration_2018-02-16.pdf
-
+serial,name,value,notes
+364,CC_calibration_date,1.518739200000E+12,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_364_Calibration_2018-02-16.pdf
+
```
Comment (cwingard) [inline calibration/DOSTAM/CGINS-DOSTAM-00364__20180216.csv:3]: Looks like there is an extra line in this file. You'll probably want to remove it.

**Review by cwingard — CHANGES_REQUESTED** [review]
OK, these all look good. As you said the other day, you are just adding the line with the salinity reference defaulted to 0. The line ending changes only affected a small hand full. They all look good with the exception of one file that had an extra blank line. I added a review comment to that file. Get that corrected, and I'm happy.

File: calibration/DOSTAM/CGINS-DOSTAM-00364__20180216.csv (hunk 1 above)
Comment (s-pearce) [reply calibration/DOSTAM/CGINS-DOSTAM-00364__20180216.csv]: okay. blank line removed.

**Review by cdobs — APPROVED** [review]
Looks good to me. Thanks @s-pearce

**Review by cwingard — APPROVED** [review]

**Pull request merged**